core/object.go

```diff
@@ -456,10 +457,22 @@ func PanicArity(n int) {
 	panic(RT.NewError(fmt.Sprintf("Wrong number of args (%d) passed to %s", n, name)))
 }
 
+func rangeString(min, max int) string {
+	if min == max {
+		return strconv.Itoa(min)
+	}
+	return strconv.Itoa(min) + ":" + strconv.Itoa(max)
```

Comment:
Maybe `from X to Y`? Or, better yet, if Y - X <= 2, something like `X, X+1 or Y`? Also, it'd be good to special case if Y = 999 (or maybe change that magic number to `MaxInt`) to handle cases like [this](https://github.com/candid82/joker/blob/master/std/os/a_os.go#L122); it could say `X or more` in this case. What do you think?